### ISSUE TITLE
padding for selected date range

### ISSUE THREAD

**Issue by hamadar** [syncfusion/flutter-widgets#1626]
![Screenshot 2024-01-11 at 11 59 53 PM](https://github.com/syncfusion/flutter-widgets/assets/79111939/e1ee2504-547a-4d7d-afdf-b18076b2cd9f)
I need to recenter the circle on the date

**Comment by Yuvaraj-Gajaraj** [comment]
We have tried to reproduce the reported issue that padding was added to the Selected date range, with the help of a shared screenshot, but the reported issue is not getting reproduced at our end. So, we kindly request you to try to replicate the issue in the shared sample below. It will be more helpful to us to provide a solution sooner.

Screenshot:


![image](https://github.com/syncfusion/flutter-widgets/assets/61863686/679b8045-b22a-4b8c-bd57-5400e483c0c1)


Sample: [i542480.zip](https://github.com/syncfusion/flutter-widgets/files/13949458/i542480.zip)
